rcl/test/rcl/test_node.cpp

```diff
@@ -94,6 +94,7 @@ TEST_F(CLASSNAME(TestNodeFixture, RMW_IMPLEMENTATION), test_rcl_node_accessors)
   {
     EXPECT_EQ(RCL_RET_OK, rcl_init_options_fini(&init_options)) << rcl_get_error_string().str;
   });
+  EXPECT_EQ(RCL_RET_OK, rcl_init_options_set_domain_id(&init_options, 42));
```

Comment:
minor comment: I'd use ASSERT_EQ to check the API instead, though i do understand this never fails and it is not something critical.

Reply:
why? IMHO, `EXPECT` should be used everywhere, except it's clear that not satisfying the condition will lead to a segmentation fault. e.g.:
```cpp
ASSERT_NE(nullptr, pointer);
EXPECT_EQ(SOME_VALUE, *pointer);
```
In that case you obviously need an `ASSERT`.

Reply:
> In that case you obviously need an ASSERT.

that makes sense to me.

i was just thinking what if `rcl_init_options_set_domain_id` fails, then it proceeds the rest of test. in that case, it would lead un-expected failure.